.github/workflows/pypi.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and upload PyPI package
+
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Docker Images
```

Comment:
Shouldnt it be "Build python package" or something similar?

Reply:
Right, fixed!